store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9640,12 +9640,12 @@ TODO: delete them permanently from here
   <globalConfigValue>1518163473</globalConfigValue>
 </attr>
 
-<attr id="3024" name="zimbraFeatureResetPasswordEnabled" type="boolean" cardinality="single" optionalIn="account,cos,domain" flags="accountCosDomainInherited,accountInfo" since="8.8.9">
-  <defaultCOSValue>FALSE</defaultCOSValue>
-  <desc>whether or not password reset feature is enabled</desc>
+<attr id="3024" name="zimbraFeatureResetPasswordStatus" type="enum" value="enabled,suspended,disabled" cardinality="single" optionalIn="account,cos,domain" flags="accountCosDomainInherited,accountInfo" since="8.8.9">
```

Comment:
Just a note... Some of these attribute IDs conflict with ones we are currently using in the [zm-mailbox develop branch in ZimbraOS](https://github.com/ZimbraOS/zm-mailbox/blob/develop/store/conf/attrs/zimbra-attrs.xml), so if we will have to pull this code into the _ZimbraOS_ repos will have to handle that.

Reply:
changed the ids to resolve conflict.